Tasks/Ananich/Task_2.py

```diff
@@ -0,0 +1,21 @@
+import re
```

Comment:
An where is it? Where's the regular expressions usage???) I had so much hope after reading the first line)